CONTRIBUTING.md

```diff
@@ -40,6 +40,13 @@ mvn verify
 
 After a successful build the RAPIDS Accelerator jar will be in the `dist/target/` directory.
 
+### Building against different CUDA Toolkit versions
+
+You can build against different versions of the CUDA Toolkit by using one of the following profiles:
+* `-Pcuda10-1` (CUDA 10.1, default)
```

Comment:
is it worth documenting these profiles consisting of a single property or should we just document the property `-Dcuda.version`?

Reply:
My original goal was to get IntelliJ to run tests using a different cuda version, adding profiles seems to be the only way to get that to work.

Reply:
I thought we could pass any property you want to a maven runner in intellij but this is probably easier